bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -1182,4 +1183,14 @@ public Item remove(String itemName, boolean recursive) {
 
     }
 
+    @Override
+    public void addRegistryHook(RegistryHook<Item> hook) {
+        throw new UnsupportedOperationException("This registry does not support hooks");
```

Comment:
Wouldn't it be an option to simply delegate it to itemRegistry, as it is done for most tother overridden methods?

Reply:
yes